Documentation/client.md

```diff
@@ -18,7 +18,7 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](.
 1. Create a new pool and volume image (10MB)
 
     ```bash
-    rookctl pool create --name rbd
+    rookctl pool create --name rbd --replica-count 1
```

Comment:
should this be a required parameter then?  or have a non-zero default?

Reply:
You can specify either erasure coding or replication. A default of 1 replica would make sense if neither is specified, but since `rookctl` is deprecated, it didn't seem worth changing the code.